### PR TITLE
new pic & music ressources, sandbox is now explore, stay home is now …

### PR DESCRIPTION
**Pull request by maccxs**
…stay safe. 

added key for full screen

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -176,17 +186,42 @@ <h5 class="card-title"> Erholter Emoji & Spieler,</h5>
...
+</script>
   <div class="card">
     <div class="card-body">
-     <h5 class="card-title"> Game Play </h5>
+     <h5 id="gameplay" class="card-title"> Game Play </h5>
            <p class="card-text">
               <ul>
             <li>Lenke den Spieler durch die Benutzung der Pfeiltasten auf der Tastatur.</li>
             <li>Q-Taste beendet das Level.</li>
+            <li>F-Taste startet Vollbildmodus.</li>
```
Comment (holke) [inline index.html]:
```suggestion
            <li>F-Taste startet den Vollbildmodus.</li>
```

**Comment by holke** [comment]
I get this with Internet Explorer:

grafik

**Comment by holke** [comment]
Firefox and edge work fine.

**Comment by maccxs** [comment]
With internet explorer all hope is lost

**Comment by holke** [comment]
Does not seem to be related to this branch.

But it works fine for youvsvirus.de but not for the TEST homepage.

**Pull request merged**